### DISCUSSION TITLE
Add terminal inside Podman Desktop

### DISCUSSION THREAD

**Discussion by trymeouteh** [podman-desktop/podman-desktop#3894] [Ideas]
VSCode has a terminal built into the app. It would be useful to have multiple terminal tabs built into Podman Desktop.

**Comment by benoitf** [comment]
There is already a terminal widget but it's not multi-tabs but you have one terminal tab in container's details
Do you connect multiple times to the same container ?

**Comment by afbjorklund** [comment]
Maybe feature is intended for `podman machine ssh` ?

* https://github.com/containers/podman-desktop/issues/889

**Reply by benoitf** [reply]
I'm wondering if the idea is to have a Terminal page where you can connect to all containers that you want vs having to go to each details page of a container to get a terminal

Like another way to see all terminals at a single location.

**Reply by deboer-tim** [reply]
+1. Would be nice to have one view that maybe started with active/in-use terminals in tabs, with simple controls to (e.g.) pick any running container and open its terminal too.

**Comment by trymeouteh** [comment]
I am referring to the terminal for each container, but a terminal at the bottom that accesses the host machine

**Reply by acancell-redhat** [reply]
+1
